RELEASE/scripts/autoscend/auto_pre_adv.ash

```diff
@@ -226,6 +226,18 @@ boolean auto_pre_adventure()
 		}
 	}
 
+	if(auto_FireExtinguisherCombatString(place) != "" || $locations[The Goatlet, Twin Peak, The Hidden Bowling Alley, The Hatching Chamber, The Feeding Chamber, The Royal Guard Chamber] contains place)
+	{
+		// force extinguisher to be equiped to overrule Slay the Dead skill forcing of a sword
```

Comment:
1. maybe adjust slay the dead to not force it if we plan to force fire extinguisher?
2. we should track whether we used the special skill in those places and only force it if we have not yet done so. Since the skills are 1 use per ascension. and we adventure more than once in those places.

Reply:
Good idea, did it. Checked code for all sources of forced weapon equips. Slay the dead was the only one that competed for a zone the extinguisher wants to be used for as well